docs/generated/galasactl_runs_delete.md

```diff
@@ -0,0 +1,31 @@
+## galasactl runs delete
+
+Get the details of a test runname which ran or is running.
+
+### Synopsis
+
+Get the details of a test runname which ran or is running, displaying the results to the caller.
+
+```
+galasactl runs delete [flags]
```

Comment:
Swap `[flags]` for `--name {run-name}`

Reply:
On second look, this auto-generated doc is valid because `galasactl runs delete` has two flags at the moment (`--help` and `--name`)